.github/workflows/functests.yml

```diff
@@ -0,0 +1,14 @@
+name: Run Functional Tests
```

Comment:
can we omit `Run` from the title in both, and just have it show up as "Functional Tests" and "Unit Tests"